src/utils/pairSearch.ts

```diff
@@ -64,6 +64,13 @@ export const createPairMaps = (
   return { pairMap, nameMap };
 };
 
+const specialCharToRomanMap: Record<string, string> = {
+  '₿': 'b',
+  '₮': 't',
+};
+const specialStringToRoman = (value: string) =>
+  value.replace(/./g, (char) => specialCharToRomanMap[char] || char);
+
```

Comment:
Personally I would have done it in a 
```suggestion
const replaceSpecialCharacters = (value: string) => {
  return value.replace(/(₿)|(₮)/g, (match, p1, p2) => {
    if (p1) return 'b';
    if (p2) return 't';
  });
}
```